lib/csv/parser.rb

```diff
@@ -928,9 +928,11 @@ def parse_no_quote(&block)
         if line.empty?
           next if @skip_blanks
           row = []
+          @quoted_fields = []
```

Comment:
How about passing `quoted_fields` to `#emit_row` and make this a local variable?

Reply:
Thanks.
This is simpler and localized !